scylla-cql/src/frame/value_tests.rs

```diff
@@ -1079,7 +1079,7 @@ fn vec_value_list() {
     );
 }
 
-fn col_spec(name: &str, typ: ColumnType) -> ColumnSpec {
+fn col_spec(name: &str, typ: ColumnType<'static>) -> ColumnSpec {
     ColumnSpec {
         table_spec: TableSpec::owned("ks".to_string(), "tbl".to_string()),
```

Comment:
This could be done with any lifetime-generic type that we have in our codebase, and there are quite a number of them. Should we introduce such an alias for each of them?
I'm a bit reluctant about type aliases, because they tend to hide that the underlying type is the same **yet** differs wrt ownership. The thing is, we want to expose that fact.